pygmt/src/xyz2grd.py

```diff
@@ -39,10 +39,10 @@ def xyz2grd(data=None, x=None, y=None, z=None, **kwargs):
     Create a grid file from table data.
 
     Reads one or more tables with *x, y, z* columns and creates a binary grid
-    file. xyz2grd will report if some of the nodes are not filled in with
-    data. Such unconstrained nodes are set to a value specified by the user
-    [Default is NaN]. Nodes with more than one value will be set to the mean
-    value.
+    file. :func:`pygmt.xyz2grd` will report if some of the nodes are not filled
```

Comment:
Not sure if this reference is necessary since it self refers to the current site.

Reply:
Yes, this is correct. I mainly changed it because of the highlighting.

Reply:
ok